spec.md

```diff
@@ -137,22 +137,37 @@ For changes in older versions, see the [release notes](https://github.com/opense
 
 ## API Version Header
 
-Requests from the Platform to the Service Broker MUST contain a header that
-declares the version number of the Open Service Broker API that the Platform
-is using:
+This specification defines an HTTP header that MUST be included in all
```

Comment:
As written this seems like a backward-incompatible change, since brokers will have to include the header on all responses.

Is that intentional?